frontend/src/metabase/dashboard/components/DashCard/DashCardParameterMapper/DashCardCardParameterMapper.jsx

```diff
@@ -94,8 +95,8 @@ function DashCardCardParameterMapper({
   const onlyAcceptsSingleValue =
     isVariableTarget(target) && !isDateParameter(editingParameter);
   const isDisabled = mappingOptions.length === 0;
-  const selectedMappingOption = _.find(mappingOptions, o =>
-    _.isEqual(o.target, target),
+  const selectedMappingOption = _.find(mappingOptions, option =>
+    _.isEqual(normalize(option.target), normalize(target)),
```

Comment:
This uses [CLJS's `normalize`](https://github.com/metabase/metabase/blob/master/frontend/src/metabase-lib/queries/utils/normalize.js#L1) which is supposed to work the same as in the BE.

[From this conversion](https://metaboat.slack.com/archives/CKZEMT1MJ/p1677782600611839), Cam suggested that FE doesn't produce the expression option, since BE doesn't use it anyway. Also, a `normalize` function could be used to achieve that.

i.e.
Use this: `[:expression "CC"]`
instead of this: `[:expression "CC" nil]`

I was about to remove the expression option from the FE but FE introduced Expression options back in:
1. [Move options handling from FieldDimension to the parent class, Dimension](https://github.com/metabase/metabase/pull/20690)
2. And [Upgrade MBQL of expression & aggregation dimension to include options](https://github.com/metabase/metabase/pull/20713)

So, [the decision has been made](https://metaboat.slack.com/archives/C04CV7H4YF7/p1677850280795319?thread_ts=1677790773.923219&cid=C04CV7H4YF7) to use `normalize` here instead for now, and this will go away when Metabase Lib v2 lands.